model/lifecycle.go

```diff
@@ -837,15 +837,6 @@ func createTasksForBuild(creationInfo TaskCreationInfo) (task.Tasks, error) {
 		for _, et := range dt.ExecTasks {
 			execTaskId := execTable.GetId(creationInfo.Build.BuildVariant, et)
 			if execTaskId == "" {
-				grip.Error(message.Fields{
-					"message":                     "execution task not found",
-					"variant":                     creationInfo.Build.BuildVariant,
-					"exec_task":                   et,
-					"available_tasks":             execTable,
```

Comment:
Why don't we keep the grip log in and just remove this line?

Reply:
If we did that we'll suddenly get a flood of errors every time server runs a generate.tasks command, so I removed it. Also fine to downgrade the level of the log though

Reply:
Hmm, I see. If they are not actionable and don't provide much value, it's probably better not to log it if we know it will be spammed. 

I feel like it would be  good to add a comment here explaining why we are not exiting or logging, but it probably shouldn't be specific to server. If you can figure out a way to explain it in a way that's generic enough, I'd advocate for that. But I'll leave it up to you.